src/baldr/graphreader.cc

```diff
@@ -34,7 +34,7 @@ GraphReader::tile_extract_t::tile_extract_t(const boost::property_tree::ptree& p
           auto id = GraphTile::GetTileId(c.first);
           tiles[id] = std::make_pair(const_cast<char*>(c.second.first), c.second.second);
         } catch (...) {
-          // skip files we dont understand
+          LOG_WARN("Tile path " + c.first + " not understood as a valhalla routing tile, skipping");
```

Comment:
sorry @PureW and @danpat i saw this question but didnt answer earlier. the point of this was sometimes we cram some other artifacts of tile building in with the tiles taht arent tiles. like connectivity map or the tz database or the admin database. warn is fine of course, just the reason i didnt put it there in the first place is that we dont really care if you put other stuff in your tile extract if you want to for convenience

Reply:
Hmm, that's kinda what I thought, I should've just asked.

I'll roll back this change, but improve the comment for our descendants.